efficientdet/hparams_config.py

```diff
@@ -222,8 +222,7 @@ def default_detection_configs():
   h.weight_decay = 4e-5
   # use horovod for multi-gpu training. If None, use TF default.
   h.strategy = None  # 'tpu', 'horovod', None
-  # precision: one of 'float32', 'mixed_float16', 'mixed_bfloat16'.
-  h.precision = None  # If None, use float32.
+  h.mixed_precision = False  # If False, use float32.
```

Comment:
Isn't precision is good enough? Do we need this extra flag? (it seems to introduce some extra code complexity)

Reply:
According to https://github.com/google/automl/issues/488.
TPUs don't support float16?